pkg/autoscaler/collector_test.go

```diff
@@ -217,11 +223,22 @@ func TestMetricCollectorRecord(t *testing.T) {
 	// After this the concurrencies are calculated correctly.
 	coll.Record(metricKey, outdatedStat)
 	coll.Record(metricKey, stat)
-	if stable, panic, err := coll.StableAndPanicConcurrency(metricKey, now); stable != panic || stable != want || err != nil {
-		t.Errorf("StableAndPanicConcurrency() = %v, %v, %v; want %v, %v, nil", stable, panic, err, want, want)
+	stable, panic, err := coll.StableAndPanicConcurrency(metricKey, now)
+	if err != nil {
+		t.Fatalf("StableAndPanicConcurrency: %v", err)
 	}
-	if stable, panic, err := coll.StableAndPanicRPS(metricKey, now); stable != panic || stable != want || err != nil {
-		t.Errorf("StableAndPanicRPS() = %v, %v, %v; want %v, %v, nil", stable, panic, err, want, want)
+	// Scale to the window sizes.
+	wantS := want / 60
+	wantP := want / 6
+	if math.Abs(stable-wantS) > 0.0001 || math.Abs(panic-wantP) > 0.0001 {
```

Comment:
create a constant `zero = 0.0001` maybe?

Or even better: Shall we add a function like

```golang
func effectivelyEqual(a, b float64) bool {
  return math.Abs(a-b) > 0.0001
}

Reply:
Added a constant. We have more FP comparisons, so we should just clean them in one go.